runtime/doc/eval.txt

```diff
@@ -1763,8 +1763,8 @@ cursor( {lnum}, {col} [, {off}])
 cursor( {list})			Number	move cursor to position in {list}
 deepcopy( {expr} [, {noref}])	any	make a full copy of {expr}
 delete( {fname})		Number	delete file {fname}
-dictwatcheradd({dict}, {pattern}, {callback})	Start watching a dictionary
-dictwatcherdel({dict}, {pattern}, {callback})	Stop watching a dictionary
+dictwatcheradd( {dict}, {pattern}, {callback})	Start watching a dictionary
+dictwatcherdel( {dict}, {pattern}, {callback})	Stop watching a dictionary
```

Comment:
@tarruda I missed this, but you have invalid formatting here: missing return type (Number I guess) and “Start/Stop watching a dictionary” should be on the next line.